manifests/cli.pp

```diff
@@ -66,8 +68,8 @@
   exec { 'safe-restart-jenkins':
     command     => "${cmd} safe-restart && /bin/sleep 10",
     path        => ['/bin', '/usr/bin'],
-    tries       => 10,
-    try_sleep   => 2,
```

Comment:
The default value of `$::jenkins::cli_try_sleep` is `10` instead of `2`.
I suppose it is a simple `buxfix`